### PR TITLE
Use std::size_t for NNCs during pinch processing.

### PR DESCRIPTION
**Pull request by blattms**

### CODE REVIEW

**Comment by blattms** [comment]
jenkins build this please

**Review by atgeirr — APPROVED** [review]
No problem with this, merge when green. What is the motivation?

**Comment by atgeirr** [comment]
jenkins build this please

File: opm/grid/cpgrid/processEclipseFormat.cpp [hunk 1]
```diff
@@ -63,7 +63,7 @@
 namespace Dune
 {
 
-    using NNCMap = std::set<std::pair<int, int>>;
+    using NNCMap = std::set<std::pair<std::size_t, std::size_t>>;
```
Comment (atgeirr) [inline opm/grid/cpgrid/processEclipseFormat.cpp:66]: You have to change this in CpGridData.hpp as well, processEclipseFormat() declaration.

**Pull request closed without merging**